Backend/Tournament_Admin/TournamentAdminService/src/main/java/TournamentAdminService/controller/MatchupController.java

```diff
@@ -0,0 +1,36 @@
+package TournamentAdminService.controller;
+
+import TournamentAdminService.dto.GameResultRequest;
+import TournamentAdminService.model.Matchup;
+import TournamentAdminService.service.MatchupService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/api/matchups")
```

Comment:
could u provide documentation on what the call looks like

Reply:
okay i will add that in for my next update